### PR TITLE
Remove need for alternative build script.

### PR DESCRIPTION
**Pull request by rlazo**

### CODE REVIEW

File: alternative-settings.gradle [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Google LLC
```
Comment (vkryachko) [inline alternative-settings.gradle:1]: Can we remove this file as well? looks like it's not needed anymore

Comment (rlazo) [reply alternative-settings.gradle:1]: Not yet.

File: build.gradle [hunk 2]
```diff
@@ -50,12 +51,19 @@ ext {
     robolectricVersion = '4.3.1'
     protocVersion = '3.14.0'
     javaliteVersion = '3.14.0'
+
+    gMavenRoot = 'https://dl.google.com/dl/android/maven2'
+    firebaseDefaultPreguardFile='oss/default-preguard.txt'
```
Comment (vkryachko) [inline build.gradle:56]: does this path exist? I can't see the oss directory in the repo

Comment (rlazo) [reply build.gradle:56]: Removed

**Comment by rlazo** [comment]
Friendly ping.

**Pull request merged**